Implement code-value pair extraction and mapping

jas88 opened the pull request:
Implement ICD11 code and meaning capture, mapping

jas88 committed:
Update DicomSource.cs

Implement ICD11 code and meaning capture, mapping

jas88 committed:
Update DicomSource.cs

Implement simple meanings + values squashing plus ICD11 extraction

rkm reviewed (changes requested):
Looks fine, but could you add a couple of basic tests?

jas88 committed:
Merge branch 'develop' into feature/sequenceflatten

jas88 committed:
Add initial SR flattener test code

Commit:
Run tests from bash so they actually test things

jas88 commented:
Also includes fix to Github actions for #416

jas88 committed:
Set a filename column value in SR flatten test

jas88 committed:
Merge branch 'feature/sequenceflatten' of https://github.com/SMI/RdmpDicom into feature/sequenceflatten

Commit:
Run test before packaging

jas88 committed:
Merge branch 'develop' into feature/sequenceflatten

jas88 committed:
Update DicomSourceUnitTests.cs

Implement some basic testing (check presence of the known ICD11 code and meaning, dermatofibroma)

jas88 committed:
Bump HIC.DicomTypeTranslation and HIC.BadMedicine.Dicom manually since Dependabot is on the blink

jas88 committed:
More SynthEHR rename baggage

jas88 committed:
Adjust .nupkg/.rdmp packaging to avoid RDMP errors

jas88 committed:
Fix typo - zip for nupkg

jas88 committed:
Merge branch 'develop' into feature/sequenceflatten

jas88 committed:
Merge branch 'develop' into feature/sequenceflatten

jas88 committed:
Update DicomSource.cs

Tidy string comparison (backslashes as ordinal not culture-specific), use LINQ for flatten query.

jas88 commented:
@rkm Not an exciting test but now has a simple SR with two levels of info on a Dermatofibrosis diagnosis, and checks that makes it into the resulting DataTable
